src/injected/analyzers/focus-traps-handler.ts

```diff
@@ -43,4 +47,12 @@ export class FocusTrapsHandler {
 
         return result;
     };
+
+    private getCurrentFocusedElement = (element: Element | null): HTMLElement => {
+        let activeElement = element as HTMLElement;
+        while (activeElement?.shadowRoot && activeElement.shadowRoot.activeElement) {
```

Comment:
Can the `while` test be simplified to `activeElement?.shadowRoot?.activeElement`?